### ISSUE TITLE
AVT2: The focus will be lost after it is moved from tooltip of Priority via keyboard on Create new notification page

### ISSUE THREAD

**Issue by pankaj-kumar34** [carbon-design-system/carbon#6841]
=====ENVIRONMENT=====
Browser: chrome
IBM CI 162 Checkpoint Violation: IBM 2.1.1 Keyboard
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/keyboard.html

=====DEFECT DESCRIPTION=====
The focus will be lost after it is moved from tooltip of Priority on Create new notification page

=====STEPS TO REPRODUCE=====
On "Tab" press, when focus reaches to tooltip then Press "Tab" again and the focus is lost, it's not going to "Priority" dropdown list.
![image](https://user-images.githubusercontent.com/18532515/93298099-42141900-f810-11ea-82ac-eb44035f2d82.png)

**Comment by emyarod** [comment]
can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox) for easier testing and debugging? it's difficult to determine where the focus is going from just the screenshot

**Comment by pankaj-kumar34** [comment]
It's fixed with latest version of carbon component.
